Upgrade che with cli (#931)
codenvy/che-docs: src/main/pages/che-7/overview/proc_preparing-openshift-3-for-installing-che.adoc

// installing-che-on-openshift-3-using-the-operator

[id="preparing-openshift-3-for-installing-che_{context}"]
= Preparing OpenShift 3 for installing {prod-short}

.Prerequisites

* A running instance of OpenShift 3.11.
* Administrator rights on this OpenShift 3 instance.

> * The `oc` OpenShift 3.11 CLI management tool is installed and configured. See link:https://access.redhat.com/documentation/en-us/openshift_container_platform/3.11/html/cli_reference/cli-reference-get-started-cli#installing-the-cli[Installing the OpenShift 3.11 CLI].

Reviewer: I think we should link to docs.openshift.com
```suggestion
* The `oc` OpenShift 3.11 CLI management tool is installed and configured. See link:https://docs.openshift.com/container-platform/3.11/cli_reference/get_started_cli.html#installing-the-cli[Installing the OpenShift 3.11 CLI].
```

* The `chectl` CLI management tool is installed. See link:{site-baseurl}che-7/installing-the-chectl-management-tool/[Installing the `chectl` CLI management tool].

.Procedure

. Log in to OpenShift. See link:https://access.redhat.com/documentation/en-us/openshift_container_platform/3.11/html/cli_reference/cli-reference-get-started-cli#basic-setup-and-login[Basic Setup and Login].
+
[subs="+attributes,+quotes",options="nowrap"]
----
$ oc login
----

> . Run following command to verify the version of the `oc` OpenShift CLI management tool is 3.11.

Reviewer: ```suggestion
. Run the following command to verify that the version of the `oc` OpenShift CLI management tool is 3.11:
```

+
[subs="+attributes,+quotes",options="nowrap"]
----
$ oc version
oc v3.11.0+0cbc58b
----

> . Run following commands to create a dummy project to find the URL that this OpenShift instance is using to deploy applications.

Reviewer: ```suggestion
. Run the following commands to create a dummy project to find the URL that this OpenShift instance is using to deploy applications.
```

+
[subs="+attributes,+quotes",options="nowrap"]
----
$ oc new-project hello-world
$ oc new-app centos/httpd-24-centos7~https://github.com/openshift/httpd-ex
$ oc expose svc/httpd-ex
$ oc get route httpd-ex
NAME     HOST/PORT                                                PATH     SERVICES PORT     TERMINATION WILDCARD
httpd-ex httpd-ex-hello-world.apps.rhpds311.openshift.opentlc.com httpd-ex          8080-tcp             None
----

> . Extract the domain from `++httpd-ex-hello-world.apps.rhpds311.openshift.opentlc.com++`. It is the part after the first name: `apps.rhpds311.openshift.opentlc.com`. Remember this URL as `<OPENSHIFT_APPS_URL>`.

Reviewer: ```suggestion
. Extract the domain from `++httpd-ex-hello-world.apps.rhpds311.openshift.opentlc.com++`. It is the part after the first name: `apps.rhpds311.openshift.opentlc.com`. Remember this URL as `__<OPENSHIFT_APPS_URL>__`.
```
